### PR TITLE
feat: inline activity feed components added

### PR DESCRIPTION
**Pull request by roger-in-kiva**
Screenshot 2024-03-08 at 12 16 04

Screenshot 2024-03-08 at 12 16 42

### CODE REVIEW

File: @kiva/kv-components/vue/KvInlineActivityCard.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<template>
+	<div
+		class="tw-flex tw-items-center tw-gap-1 tw-py-0.5 tw-px-1 tw-shadow-lg"
+		style="border-radius: 8px;"
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvInlineActivityCard.vue:4]: Is there a tailwind class that's close enough?

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvInlineActivityCard.vue:4]: Or change to 4px, could make more sense for that card.

Comment (dyersituations) [reply @kiva/kv-components/vue/KvInlineActivityCard.vue:4]: If 4px seems to make sense, go for it. We can generally use our existing styles in Tailwind.

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvInlineActivityCard.vue]: Very biased sometimes...

**Review by dyersituations — APPROVED** [review]

**Pull request merged**